shop-be/product-service/src/functions/getProductById/handler.ts

```diff
@@ -1,21 +1,21 @@
 import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/apiGateway';
 import { formatJSONResponse } from '@libs/apiGateway';
+import { dbConnectAndExecute } from '@libs/db';
 import { middyfy } from '@libs/lambda';
-import productsList from '../../resources/productList.json';
+import { Product } from 'src/types/product';
 
 import schema from './schema';
 
 const getProductById: ValidatedEventAPIGatewayProxyEvent<typeof schema> = async (event) => {
   const { pathParameters: { id } } = event;
-  const product = productsList.find(item => item.id === id);
+  const { rows: product } = await dbConnectAndExecute<Product>({ query: 'select id, title, description, price, count from products p left join stocks s on p.id = s.product_id where id = $1;', params: [id] })
```

Comment:
As we discussed about a structure of the lambda - it's good to move db logic to the separate service (tight coupling now - handler know about data source).

Reply:
moved